### PR TITLE
ADEN-6976 Add device info to kikimora tracking

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/tracking/adInfoTracker.js [hunk 1]
```diff
@@ -45,6 +46,7 @@ define('ext.wikia.adEngine.tracking.adInfoTracker',  [
 			'pv': pageParams.pv || '',
 			'pv_unique_id': win.pvUID,
 			'browser': [ browserDetect.getOS(), browserDetect.getBrowser() ].join(' '),
+			'device': megaAdUnitBuilder.getDevice(pageParams),
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/tracking/adInfoTracker.js]: I believe we don't want to have results like `unknown-specialpage` :P. Can we avoid using megaAdUnitBuilder for that? What about extracting separate module for device and use it here and in megaAdUnitBuilder?

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/tracking/adInfoTracker.js]: Done!

File: extensions/wikia/AdEngine/js/utils/device.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* global define */
+define('ext.wikia.adEngine.utils.device', [
+	'wikia.browserDetect'
+], function (browserDetect) {
+	'use strict';
+
+	var result = '';
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/utils/device.js]: We can set 'unknown' by default

File: extensions/wikia/AdEngine/js/utils/device.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+	function getDevice(params) {
+		if (params.skin === 'oasis') {
+			result = browserDetect.isMobile() ? 'tablet' : 'desktop';
+		} else if (params.skin === 'mercury' || params.skin === 'mobile-wiki') {
+			result = 'smartphone';
+		}
+
+		return result;
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/utils/device.js]: return 'unknown' please

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/utils/device.js]: Done.

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**